hoomd/hpmc/pytest/test_quick_compress.py

```diff
@@ -35,8 +50,17 @@
     ('trigger', hoomd.trigger.Periodic(10000)),
     ('trigger', hoomd.trigger.After(100)),
     ('trigger', hoomd.trigger.Before(12345)),
-    ('target_box', hoomd.Box.from_box([10, 20, 30])),
-    ('target_box', hoomd.Box.from_box([50, 50])),
+    ('target_box', hoomd.variant.box.Constant(hoomd.Box.from_box([10, 20,
```

Comment:
Should I be able to set the target_box attribute to a regular box too?

Reply:
Yes, you need to be able to set a regular box both for backwards compatibility and convenience.

Reply:
Agreed with Josh.

Reply:
A test case still needs to be added for this.

Reply:
You're right. Added.